plaid-stl/src/github/mod.rs

```diff
@@ -241,6 +241,110 @@ pub fn remove_user_from_repo_detailed(repo: &str, user: &str) -> Result<(), Plai
     Ok(())
 }
 
+/// List all seats in org's Copilot subscription
+/// ## Arguments
+///
+/// * `org` - The org owning the subscription
+/// * `per_page` - The number of results per page (max 100)
+/// * `page` - The page number of the results to fetch.
+pub fn list_copilot_subscription_seats(org: &str, per_page: Option<u64>, page: Option<u64>) -> Result<String, PlaidFunctionError> {
```

Comment:
I'm not averse to having this function but I think generally speaking people will want to pull all of them so at least having a helper function to do that is going to prevent code duplication in modules.